js/main.js

```diff
@@ -741,32 +742,47 @@ function doTasks() {
     // Legacy
     doTask(gameData.currentSkill);
     doTask(gameData.currentBattle);
+    doTask(gridStrength);

```

Comment:
muss das nicht auch iwie in die gameData rein, weils sonst nicht gespeichert wird?

Reply:
ja, die gameData hat halt auch nur die referenz aufs selbe singleton. daher so schöner?